addons/pos_restaurant/views/pos_config_views.xml

```diff
@@ -57,7 +57,7 @@
                         <field name="is_order_printer"/>
                     </div>
                     <div class="o_setting_right_pane">
-                        <label for="is_order_printer"/>
+                        <label for="is_order_printer" string="Order Printers"/>
```

Comment:
Same question: Is this really done to avoid a name clash in the export wizard?

Reply:
yes